lerna.json

```diff
@@ -1,6 +1,7 @@
 {
   "packages": [
```

Comment:
I think github actions use lerna - not sure though. Doesn't it just run `npm run test` (which uses lerna)?

Reply:
Ok I checked the build for this PR and it looks like Github is good to go:

lerna info Executing command in 4 packages: "npm run test"
lerna info run Ran npm script 'test' in 'harmony-work-scheduler' in 42.2s:

> harmony-work-scheduler@1.0.0 test
> eslint --ext .ts . && nyc mocha --timeout 5000 && better-npm-audit audit

Although we shouldn't default the timeout to 5 seconds for this one - I'm guessing that was an artifact of copying from service-runner .mocharc.yml.

We will likely still need to change Bamboo for building and deploying the images since those are not done through `lerna`.

Reply:
Removed the 5 second default timeout.

Reply:
Make sure to update the Bamboo deployments to build and deploy the images like we do with the tasks when you merge this in.